users/models.py

```diff
@@ -274,3 +275,41 @@ def create_last_seen_for_user(sender, instance, created, raw, **kwargs):
 
     if created:
         LastSeen.objects.create(user=instance, at=timezone.now())
+
+
+class InvitationToken(models.Model):
```

Comment:
I couldn't find in the current code whether there is some other use for tokens (password resets, email confirmation, etc).

If so, would it be worth considering some unification of those classes?